### DISCUSSION TITLE
Surface chemical reaction

### DISCUSSION THREAD

**Discussion by DrDrunkstein** [jax-ml/jax#24270] [General]
First of all, thank you for the great work on this project! I’ve been exploring various open-source possibilities, and this repository seems to be a very promising solution for what I’m looking to achieve.

I’m interested in implementing a _reaction_ that occurs on the _surface_ of the mesh, and from my understanding, this might involve using a Robin boundary condition. However, I’m not entirely certain that this is the right approach.

Could you provide some guidance? Is the Robin boundary condition suitable for modeling surface reactions in this context, or would you recommend a different approach within the XLB framework?

Thanks!

**Comment by jakevdp** [comment]
Hi - this seems like a domain-specific question, and would be better asked in a place where domain experts might see it. Perhaps you might post the question at https://github.com/Autodesk/XLB?

**Reply by DrDrunkstein** [reply]
Thank you, will do!
